### ISSUE TITLE
Make the game detect the raw files aswell.

### ISSUE THREAD

**Issue by TheHopelessThrumbo** [OpenRA/ra2#592]
Installing RA2 from the original installation disk is impossible now, I ran the game from the raw files since I use Windows 7. However, OpenRA is looking for the full installation, without success. Thus, I'd like to be able to just load the raw game files, which are in full integrity and working.

**Comment by abcdefg30** [comment]
Hm, I suppose we can mention where a user can put the files in the wiki. Place `language.mix`, `ra2.mix` and `theme.mix` inside the ra2 content folder in OpenRA's support folder: `OpenRA\Content\ra2`. The support folder is in your `My Documents` (and in `AppData` if it's not in docs).

**Comment by TheHopelessThrumbo** [comment]
I did so, but it should be automatic to reduce issues for less experienced people.

**Comment by Mailaender** [comment]
I don't know what the raw game files are, but you can make your own installer by configurating https://github.com/OpenRA/ra2/tree/master/mods/ra2/installer

**Comment by logpig** [comment]
where exactly do i have to put these files?
i tried: 
~/.openra/Content/ra2/
~/.OpenRA/Content/ra2/
~/Support/.openra/Content/ra2/
and i tried to put the Support folder with the content in the same directory as i installed openra2
but the game won't recognize any file...

on manjaro/unix installed with make, the normal way...

**Comment by Mailaender** [comment]
`~/.config/openra/Content/ra2/`

**Comment by Mailaender** [comment]
[fetch-content.sh](https://gist.github.com/Mailaender/1c9962667f1c453598523db8be51b9bc) may also helpful as it automatizes everything.

**Comment by logpig** [comment]
now the wiki shows the right directory...
thanks

**Comment by Mailaender** [comment]
Yeah, sorry I just realize. For backwards compatibility the old directory still works as long as the new one doesn't exist which makes it even more confusing.
